modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/result/display/context/builder/SearchResultSummaryDisplayContextBuilder.java

```diff
@@ -519,14 +521,21 @@ private void _buildAssetCategoriesOrTags(
 	}
 
 	private void _buildAssetRendererURLDownload(
-		SearchResultSummaryDisplayContext searchResultSummaryDisplayContext,
-		AssetRenderer<?> assetRenderer, Summary summary) {
+			SearchResultSummaryDisplayContext searchResultSummaryDisplayContext,
```

Comment:
Could you Sort the parameters?

Reply:
Sure! There's several `_build*` functions that needed sorted parameters, I'll update those too.